app.py

```diff
@@ -129,6 +128,7 @@ def load_adventures_per_level(lang, level):
             continue
         # end adventure is the quiz
         # if quizzes are not enabled, do not load it
+        # Todo TB -> Is this still relevant? Teachers can simply "disable" adventures in customizations!
```

Comment:
I don't think we still need this indeed! And (related) should we just call the tab "quiz" instead of end? No sure how that came to be but it is a bit weird.